src/Joomlatools/Composer/ExtensionInstaller.php

```diff
@@ -283,53 +281,4 @@ protected function _getApplicationMessages()
 
         return $descriptions;
     }
-
-    /**
-     * Enable all plugins that were installed with this package.
-     *
-     * @param PackageInterface $package
-     * @param string           $subdirectory Subdirectory in package install path to look for plugin manifests
-     */
-    protected function _enablePlugin(PackageInterface $package, $installPath, $subdirectory = '')
-    {
-        $file = false;
-        $path = realpath($installPath . '/' . $subdirectory);
-
-        if($path !== false)
-        {
-            $file = Util::getPackageManifest($path);
-        }
-
-        if($file !== false)
-        {
-            $manifest = simplexml_load_file($file);
-            $type     = (string) $manifest->attributes()->type;
-
-            if ($type == 'plugin')
-            {
-                $name  = Util::getNameFromManifest($installPath);
-                $group = (string) $manifest->attributes()->group;
-
-                $extension = Bootstrapper::getInstance()->getApplication()->getExtension($name, 'plugin', $group);
-
-                if (is_object($extension) && $extension->id > 0)
-                {
-                    $sql = "UPDATE `#__extensions`"
-                        ." SET `enabled` = 1"
-                        ." WHERE `extension_id` = ".$extension->id;
-
-                    \JFactory::getDbo()->setQuery($sql)->execute();
-                }
-            }
-            elseif ($type == 'package')
```

Comment:
@danielsmink - IMHO I think it is probably best to just remove this `elseif` block where it calls `_enablePlugin` recursively.  I believe the rest of the code works as expected when it's not being called as a package.

Reply:
I'm not sure about the use case lets see what @stevenrombauts says about it. If needed I can alter it that way.